mmengine/runner/runner.py

```diff
@@ -690,6 +690,41 @@ def set_randomness(self,
             deterministic=deterministic,
             diff_rank_seed=diff_rank_seed)
 
+    def set_torch_cfg(self, torch_cfg: dict) -> None:
+        """Set torch variable to by config.
+
+        Args:
+            torch_cfg (dict): Contains key-value pair which defines the names
+                of torch variables and corresponding values.
+
+        Examples:
+            >>> torch_cfg = dict(
+            >>>    backends.cuda.matmul.allow_tf32=True,
+            >>>    torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction=True
```

Comment:
`torch.backends....` or `backends....`?

Reply:
both are OK, since you can access `torch.torch.torch...` recursively :rofl: